Automattic/wp-calypso
# Hosting dashboard: clean up monitoring and logs tabs

Part of:

- https://github.com/Automattic/dotcom-forge/issues/7924

## Proposed Changes

This PR refactors the code for the hosting dashboard's Monitoring and Logs tabs. Previously, they were under the `client/my-sites/site-monitoring/` directory, because they were once in a single page in site-specific view (not global view).

Now, this PR cleans those up by splitting them into separate `client/hosting/monitoring/` and client/hosting/logs/` directories.

## Why are these changes being made?

Cleaning up code to avoid future confusion.

## Testing Instructions

1. Go to `/sites`.
2. Click an Atomic site.
3. Smoke test the Monitoring and Logs tabs. Make sure the features still work.

## Pre-merge Checklist
